rental/build.gradle

```diff
@@ -0,0 +1,26 @@
+plugins {
+    id 'org.springframework.boot' version '2.6.5'
```

Comment:
If possible, please add the fixes we applied earlier
after version '2.6.5' add:
apply false